packages/picasso.js/src/web/components/brush-area/brush-area.js

```diff
@@ -180,9 +182,6 @@ const definition = {
     return 0;
   },
   render() {},
-  beforeRender({ size }) {
-    this.rect = size.computedInner;
```

Comment:
Not using computedInner will likely break the component when the dock layout is scaled